### DISCUSSION TITLE
pairing Nodon sin-4-fp-21

### DISCUSSION THREAD

**Discussion by Christophebal** [Koenkk/zigbee2mqtt#25167] [Devices]
Hi, I am trying to pair a sin-4-fp-21 nodon.
the light flashes orange for around 30 seconds then turns green, but it is not seen by Zigbee2MQTT.

**Comment by Nardol** [comment]
I was in the same situation.
It turned green but nothing was added to Zigbee2MQTT, until a long delay after.
But in a first attempt, I wished to use it as a manager for all my radiators by replacing the previous manager I had so only wiring three wires instead of five : phase and neutral to power the nodon and the pilot wire.
It turned green but nothing was added to Zigbee2MQTT.
It only worked when I connected it to a radiator so wiring all the cables as it seams but after waiting for a long delay.

Hop this helps.
